.github/workflows/component-go.yaml

```diff
@@ -1,6 +1,9 @@
 name: Go
 
 on:
+  # We trigger this workflow once a week on Saturdays to make sure the examples don't get stale
+  schedule:
+    - cron: "0 0 * * 6"
```

Comment:
I'm not sure I understand what purpose this serves, how do the examples go stale?

Reply:
This is making sure that as we release new versions of wash (even patch versions) we can make sure that nothing has broken. It means we'll always be making sure they still build and run

Reply:
I want to eventually work on a larger e2e test with all the different examples, but this buys us some extra insurance

Reply:
So a couple of things:
1. We pin the version of wash that's used [in this workflow as part of the `env` block](https://github.com/wasmCloud/go/pull/53/files#diff-a5d9d7f6238b1815721439608bf5822cfffa3e654acc8107cfcec253060ab9d8L17-L21), so it only tests against a specific version of `wash`, not latest.
2. If this runs on a weekend as this is currently configured, I'm not sure there's anyone there to look at it, and given that a dependabot flurry of updates will just push this way down the Actions page (or obviate the need entirely given that those PRs _do_ run this workflow), I'm not really sure it adds value.
3. If the idea is to inform whether a bunch of wash changes potentially broke the examples in a given language, wouldn't it be better to run those tests as part of wash's own workflow?

That being the case, I think we should just remove this schedule section.